nautobot/docs/installation/region-and-site-data-migration-guide.md

```diff
@@ -0,0 +1,405 @@
+# Region and Site to Location Migration Guide
+
+In Nautobot 2.0.0, all the `Region` and `Site` related data models are being migrated to use `Location`. Below is a comprehensive guide for Nautobot users and Nautobot App developers to migrate their `Region` and `Site` related models and apps to `Location`.
+
+## Migrate ObjectPermission instances in Nautobot from Region and Site to Location
+
+Nautobot admins need to migrate `Site` and `Reigon` Related `ObjectPermission` instances to `Location`. The correct way to do it are documented below with practical examples.
+
+### Region Specific ObjectPermission
+
+Add `Location` ContentType to `object_types` field of the `ObjectPermission` if it is not already present.
+
+Since `Location` has all the filters that `Region` had and they retain the same functionalities. We do not need to modify the `constraints` field of any `Region` specific `ObjectPermission` instances.
+
+### Site Specific ObjectPermission
+
+Add `Location` ContentType to `object_types` field of the `ObjectPermission` if it is not already present.
+
+The old `constraints` field for `Site` specifc `ObjectPermission` instances might look like this:
+
+```json
+{
+    "name": "AMS01",
+    "name__in": ["AMS01", "ATL01", "EDX01"],
+    "slug": "ams01",
+    "slug__in": ["ams01", "atl01", "edx01"],
+    "id": "9121jfjnjas9fc2-0326-40db-b3a3-3af1231ad",
+    "id__in": ["9121jfjnjas9fc2-0326-40db-b3a3-3afe2207c1ad", "mklmjb79012mkjasd-skmk198721-091231"],
```

Comment:
Ok, these definitely aren't real/valid UUIDs. :-) Can you put some "real" UUIDs in here? I got distracted by seeing these nonsense strings.